tg/colors.py

```diff
@@ -2,7 +2,7 @@
 
 DEFAULT_FG = curses.COLOR_WHITE
 DEFAULT_BG = curses.COLOR_BLACK
-COLOR_PAIRS = {10: 0}
+COLOR_PAIRS = {(10, 10): 0}
```

Comment:
@paul-nameless was it as bug?

Reply:
Don't know, I think we can try to remove it and see if color scheme still works fine.